### PR TITLE
Filter menu hidden in mobile view

### LINKED ISSUE #2439: [Resources] Mobile-friendly: hide the filters menu or keep it completely collapsed
Regarding mobile-friendly view for https://layer5.io/resources, the following feedback was given by @warunicorn19:
> @debo19 could you please hide the filters menu or keep it completely collapsed in the mobile view. Right now it covers up a lot of space in mobile view. Though, we can probably work on that separately.

_Originally posted by @warunicorn19 in https://github.com/layer5io/layer5/issues/2430#issuecomment-1003046908_

### PR DESCRIPTION
**Pull request by Ciggzy1312**
Signed-off-by: Deepayan <deepayanmukherjee1312@gmail.com>

**Description**
Added hide button to keep filters hidden in mobile view
This PR fixes #2439 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit a2ec18ae031238c6b8ff6cc40349bd4b70f6e369 at: https://61d9f13e3d5c03b974ae27a0--layer5ng.netlify.app

**Comment by warunicorn19** [comment]
Package-lock will have to be reverted here.

**Comment by YashKamboj** [comment]
@Ciggzy1312 there is no button to hide the menu between 1000px and 750px, and making the default state as hidden for the menu would also be good idea.

https://user-images.githubusercontent.com/75125203/148674424-082b56e9-6c1a-4a01-9ac9-472322284d67.mp4

**Comment by Ciggzy1312** [comment]
Nice catch @YashKamboj 
Will fix it

**Comment by Nikhil-Ladha** [comment]
The current resources page is spread out way toooo much!
Maybe you could send out a different PR after this to fix that, it should look similar to `/blog` (wrt to the width)

**Comment by warunicorn19** [comment]
@Nikhil-Ladha I think the reason that it is spread out is because of the way the filters work in resources. It is a global filter. Though the code could use some refactoring.

**Comment by Nikhil-Ladha** [comment]
> @Nikhil-Ladha I think the reason that it is spread out is because of the way the filters work in resources. It is a global filter. Though the code could use some refactoring.

Wdym by "global-filter" ? Whatever data it filters it is within the page itself, so what's the issue in enclosing everything within a single "container" tag?

**Comment by warunicorn19** [comment]
Actually the filter works on blogs and news as well if I'm not wrong. @debo19 would probably be able to explain it better.

**Comment by warunicorn19** [comment]
I guess we can enclose it in a single container tag 🤔

**Comment by debo19** [comment]
> Actually the filter works on blogs and news as well if I'm not wrong. @debo19 would probably be able to explain it better.

@warunicorn19 It's a styling issue, that shouldn't be a problem. @Ciggzy1312 I'll look into this in my open PR, you can go ahead with adding the Open filter button as discussed today.

**Comment by warunicorn19** [comment]
I completely misunderstood what @Nikhil-Ladha was saying🤦‍♂️.

**Comment by l5io** [comment]
🚀 Preview for commit a88db7aa482100762850ddd7bcc4915e8d6cc2a9 at: https://61df36d2a6ffc0398dd9eec6--layer5ng.netlify.app

File: src/sections/Resources/Resources-grid/filters.js [hunk 1]
```diff
@@ -69,138 +72,155 @@ const Navigation = (props) => {
...
+          setExpandFilter(!expandFilter); 
+        }} className="filter-menu-icon">
+          {expandFilter ? (<>
+            <IoClose className="filter-icon" />
+            <span>Close Filters</span>
+          </> 
+          )  :
+            (
+              <>
+                <FiFilter className="filter-icon"/>
```
Comment (debo19) [inline src/sections/Resources/Resources-grid/filters.js:88]: open

Can you enclose this inside a Button? Also, increase the space between the resource card and the button.

File: src/sections/Resources/Resources-grid/filters.js [hunk 2]
```diff
@@ -69,138 +72,155 @@ const Navigation = (props) => {
...
-
   return (
     <ResourceNavigationWrapper>
       <div className="filter">
+        <div onClick={() => {
+          setExpandFilter(!expandFilter); 
+        }} className="filter-menu-icon">
+          {expandFilter ? (<>
+            <IoClose className="filter-icon" />
+            <span>Close Filters</span>
```
Comment (debo19) [inline src/sections/Resources/Resources-grid/filters.js:83]: clearafter

We still need the `Clear Filters` option like we have now. 

clearbefore

**Review by debo19 — CHANGES_REQUESTED** [review]

File: src/sections/Resources/Resources-grid/filters.js (hunk 1 above)
Comment (debo19) [reply src/sections/Resources/Resources-grid/filters.js:88]: Align the text to the center of the icon.

**Comment by l5io** [comment]
🚀 Preview for commit d06a61b8b41a19ec3948f024195799dbdaca1140 at: https://61e0538778d1d60062a586b4--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 91640c950e3a43cf5abf2c47c874d666f560bbbc at: https://61e10fc6a3cde2aad7bc1671--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit f5eb5d0a7e7fd118dfba55bbc8de3b50cd89b3e7 at: https://61e59c1020b5dcbf1e908eb1--layer5ng.netlify.app

**Comment by leecalcote** [comment]
Heads-up - the arrow is being line-wrapped (in Safari, at least) -
Screen Shot 2022-01-17 at 2 50 49 PM

**Pull request closed without merging**